<launch>
  <arg name="config_pendulum" doc="The configuration parameters of the pendulum for the current experiment."/>
  <arg name="config_platform" doc="The configuration parameters of the platform for the current experiment."/>
  <arg name="config_protocol" doc="The configuration parameters of the protocol for the current experiment."/>
  <arg name="config_subject" doc="The configuration parameters of the subject for the current experiment."/>

  <group ns="dysturbance">
    <!-- pendulum information -->
    <rosparam file="$(find dysturbance_tests)/config/$(arg config_pendulum).yaml" command="load"/>

    <!-- platform information -->
    <rosparam file="$(find dysturbance_tests)/config/$(arg config_platform).yaml" command="load"/>

    <!-- protocol information -->
    <rosparam file="$(find dysturbance_tests)/config/$(arg config_protocol).yaml" command="load"/>

    <!-- subject information -->
    <rosparam file="$(find dysturbance_tests)/config/$(arg config_subject).yaml" command="load"/>
    
    <node name="dysturbance_test_reemc_gazebo" pkg="dysturbance_tests" type="dysturbance_test_reemc_gazebo.py"
          respawn="false" output="screen" required="true"/>


  </group>
</launch>